tests/test_vlan.rb

```diff
@@ -316,4 +316,29 @@ def test_mapped_vnis
   rescue RuntimeError => e
     hardware_supports_feature?(e.message)
   end
+
+  def test_vlan_fabric_control
+    vlan = Vlan.new('100')
+    assert_equal(vlan.default_fabric_control, vlan.fabric_control,
+                 'Error: Vlan fabric-control is not matching')
+    vlan.fabric_control = true
+    assert(vlan.fabric_control)
+    vlan.destroy
+  end
+
+  def test_another_vlan_as_fabric_control
+    vlan = Vlan.new('100')
+    assert_equal(vlan.default_fabric_control, vlan.fabric_control,
+                 'Error: Vlan fabric-control is not matching')
+    vlan.fabric_control = true
+    assert(vlan.fabric_control)
+    another_vlan = Vlan.new(101)
+
+    assert_raises(RuntimeError,
+                  'VLAN misconfig did not raise CliError') do
+      another_vlan.fabric_control = true
+    end
+    vlan.destroy
+    another_vlan.destroy
+  end
```

Comment:
1. Your tests are okay but the second tests does the same as the first one so I would just remove the first one
2. fabric-control is not supported on some platforms so you need to explicit test for the exclusions

e.g.:

```
def test_fabric_control
    if validate_property_excluded?('vlan', 'fabric_control')
      assert_raises(Cisco::UnsupportedError) { Vlan.new('100').fabric_control = true }
      return
    end

    v100 = Vlan.new('100')
    assert_equal(v100.default_fabric_control, v100.fabric_control)
    v100.fabric_control = true
    assert(v100.fabric_control)

    v101 = Vlan.new(101)
    assert_raises(RuntimeError,
                  'VLAN misconfig did not raise CliError') do
      v101.fabric_control = true
    end
    v100.destroy
    v101.destroy
  end
```